### DISCUSSION TITLE
Problem with tabs

### DISCUSSION THREAD

**Discussion by DmiRials** [rustdesk/rustdesk#6179] [General]
If you open several connections in tabs and one of the hosts is no longer available (reboot, disconnection), then you cannot switch to another tab.
Is it possible to solve or fix this?
On version 1.2.2 it worked. But in newer versions, the described problem occurs

**Comment by rustdesk** [comment]
@fufesou

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/6181
